data/transport missions.txt

```diff
@@ -1828,6 +1828,50 @@ mission "Courier 1"
 		dialog phrase "generic cargo delivery payment"
 
 
+phrase "generic human package offer"
+	word
+		`A local`
+		`A man`
+		`A resident of <source>`
+		`A stranger`
+		`A woman`
+		`One of the spaceport's dock workers`
+	word
+		` `
+	word
+		`appears from seemingly nowhere to ask,`
+		`approaches you and says,`
+		`flags you down and asks,`
+		`walks up to you and asks,`
+	word
+		` `
+	word
+		`"Captain,`
+		`"Excuse me, Captain.`
+		`"Hello, Captain.`
+		`"Hello.`
+		`"Sorry to interrupt, but`
+	word
+		` `
+	word
+		`I have a small package that needs to get to <planet> by <day>.`
+		`I'm trying to get a package to a relative on <planet> by <day>.`
+		`one of my clients on <planet> needs this package by <day>.`
```

Comment:
Some of the previous words end in periods, meaning this word here could result in a combination like `Hello, Captain. one of my clients...`